pkg/expapi/latest/latest.go

```diff
@@ -0,0 +1,93 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package latest
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/api"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/api/meta"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/api/registered"
+	_ "github.com/GoogleCloudPlatform/kubernetes/pkg/expapi"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/expapi/v1"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/runtime"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/util"
+)
+
+var (
+	Version  string
+	Versions []string
+
+	accessor   = meta.NewAccessor()
+	Codec      runtime.Codec
+	SelfLinker = runtime.SelfLinker(accessor)
+	RESTMapper meta.RESTMapper
+)
+
+func init() {
+	Version = registered.RegisteredVersions[0]
```

Comment:
There is a lot of refactoring that should be done in general and I think that's beyond the scope of this PR.